src/Lesson1/HomeWorkApp.java

```diff
@@ -0,0 +1,35 @@
+package Lesson1;
+
+public class HomeWorkApp {
+    public static void main(String[] args) {
+threeWords();
+checkSumSign();
+printColor();
+compareNumbers();
+    }
+
+    public static void threeWords() {
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+    public static void checkSumSign() {
+        int a = 3;
+        int b = 5;
+        int sum = a + b;
+        if (sum >= 0) System.out.println("Сумма положительная");
+            else System.out.println("Сумма отрицательная");
+    }
+    public static void printColor() {
+        int value = 50;
+        if (value <= 0) System.out.println("Красный");
+        else if (value > 0 && value <= 100) System.out.println("Желтый");
```

Comment:
```suggestion
        else if (value <= 100) System.out.println("Желтый");
```
поскольку данное условие работает само по себе из-за неактульности первого `value<= 0 -> false <=> value > 0`